soda_machine.rb

```diff
@@ -1,18 +1,35 @@
+require_relative "soda.rb"
+
 class SodaMachine
   attr_reader :sodas, :cash
+  @@inventory = 0
 
   def initialize(args = {})
     @sodas = args[:sodas]
+    self.class.add_soda(@sodas.length)
     @cash = args[:cash]
   end
 
-  def current_inventory_count
+  def self.current_inventory_count
+    return @@inventory
```

Comment:
Also, what data structure (given to you) describes the sodas currently in the machine?  How can you leverage that?